rsa/key.py

```diff
@@ -752,13 +739,11 @@ def newkeys(nbits, accurate=True, poolsize=1, exponent=DEFAULT_EXPONENT):
         raise ValueError('Pool size (%i) should be >= 1' % poolsize)
 
     # Determine which getprime function to use
+    getprime_func = rsa.prime.getprimebyrange
```

Comment:
Don't assign and then re-assign three lines later. Just keep the if/else construct that was there. Unless you have a good reason to it this way (rather than personal preference), of course, but then include an explanation why it's done that way.